tests/scenarios/json_flag_rules/repo/ick.toml

```diff
@@ -0,0 +1,30 @@
+[[ruleset]]
+path = "."
```

Comment:
you can give this a prefix to force the qualname to be different in the scenario's list-rules